app/rest/rest/src/main/java/io/syndesis/rest/v1/handler/connection/CustomConnectorHandler.java

```diff
@@ -61,7 +73,33 @@ public Connector create(final ConnectorSettings connectorSettings) {
     @Consumes(MediaType.APPLICATION_JSON)
     @ApiOperation("Provides a summary of the connector as it would be built using a ConnectorTemplate identified by the provided `connector-template-id` and the data given in `connectorSettings`")
     public ConnectorSummary info(final ConnectorSettings connectorSettings) {
-        return withGeneratorAndTemplate(connectorSettings.getConnectorTemplateId(), (generator, template) -> generator.info(template, connectorSettings));
+        return withGeneratorAndTemplate(connectorSettings.getConnectorTemplateId(),
+            (generator, template) -> generator.info(template, connectorSettings));
     }
 
+    @GET
+    @Path("/")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation("Returns all Connectors that have been created from a given template identified by `templateId`")
+    @ApiImplicitParams({
+        @ApiImplicitParam(name = "sort", value = "Sort the result list according to the given field value", paramType = "query",
+            dataType = "string"),
+        @ApiImplicitParam(name = "direction",
+            value = "Sorting direction when a 'sort' field is provided. Can be 'asc' " + "(ascending) or 'desc' (descending)",
+            paramType = "query", dataType = "string"),
+        @ApiImplicitParam(name = "page", value = "Page number to return", paramType = "query", dataType = "integer", defaultValue = "1"),
+        @ApiImplicitParam(name = "per_page", value = "Number of records per page", paramType = "query", dataType = "integer",
+            defaultValue = "20")
+
+    })
+    @ApiResponses(@ApiResponse(code = 200, response = ListResult.class,
+        message = "List of connectors created from template identified by `templateId`"))
+    public ListResult<Connector> list(
```

Comment:
Would this be better just adding under `/connectors`, expanding `list` method to handle filtering too? Seems like generic connector stuff to me and means we could implement server side filtering for queries.